src/index.js

```diff
@@ -252,14 +258,13 @@ function cleanObject(obj) {
   return obj;
 }
 
-function eventProperties(e) {
-  let target = e.target;
```

Comment:
Let's use `this` for the `target` and keep the rest of the method intact, so #55 will work.

Reply:
oh I got rid of eventProperties and changed it to elementProperties as all it did was take the target from the event.

Reply:
Changed it back to eventProperties and made it clear what 'this' is.